scylla/datadog_checks/scylla/metrics.py

```diff
@@ -376,18 +376,18 @@
 
 ADDITIONAL_METRICS_MAP = {
     'scylla.alien': SCYLLA_ALIEN,
-    'scylla.batchlog': SCYLLA_BATCHLOG,
+    'scylla.batchlog_manager': SCYLLA_BATCHLOG_MANAGER,
     'scylla.commitlog': SCYLLA_COMMITLOG,
     'scylla.cql': SCYLLA_CQL,
     'scylla.database': SCYLLA_DATABASE,
-    'scylla.execution': SCYLLA_EXECUTION,
+    'scylla.execution_stages': SCYLLA_EXECUTION_STAGES,
     'scylla.hints': SCYLLA_HINTS,
     'scylla.httpd': SCYLLA_HTTPD,
-    'scylla.io': SCYLLA_IO,
+    'scylla.io_queue': SCYLLA_IO_QUEUE,
     'scylla.lsa': SCYLLA_LSA,
     'scylla.memory': SCYLLA_MEMORY,
     'scylla.memtables': SCYLLA_MEMTABLES,
-    'scylla.query': SCYLLA_QUERY,
+    'scylla.query_processor': SCYLLA_QUERY_PROCESSOR,
```

Comment:
👍 These are great, but the corresponding names in `conf.yaml.example` should be updated to match.

Reply:
Addressed in 70b50e5 :+1: